### ISSUE TITLE
Add/extend arakoon collapse check with available disk space verification for local collapse

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/openvstorage-health-check#222]
## Problem description
Recommended collapse: local
Requires: head.db size x2 minimal

We currently do not check if we will be able to run a local collapse, as disk space gets consumed and the local_collapse does not/will not be run, we might end up with a situation where we're unable to run a local collapse anyway.

Solution:
Add disk space check / partition for all arakoon configured there:

(sum of all head.dbs) x2 = minimal amount of diskspace required (when run simultaneously)

Return:
Critical: free space = <= (sum of all head.dbs) x2
Warning: free space = <= (sum of all head.dbs) x3
OK: free space = <= (sum of all head.dbs) x4

This output can be parsed by check_mk afterwards so operations can take appropriate action.

**Comment by pploegaert** [comment]
@jeroenmaelbrancke @matthiasdeblock: pls provide additional input/comments if necessary

**Comment by jeroenmaelbrancke** [comment]
please check on head.db.clone

The head.db.clone.tmp is the moment when arakoon copy the head.db to head.db.clone (temp file)
The head.db.clone.wal is from tokyo cabinet

examples:
```
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 06:37 mybackend-nsm_0/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs     0 Dec 19 04:32 mybackend-nsm_0/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  325M Dec 19 04:56 mybackend-nsm_10/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs   16K Dec 19 06:49 mybackend-nsm_10/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  872K Dec 19 04:56 mybackend-nsm_10/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  436M Dec 19 07:26 mybackend-nsm_11/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  832K Dec 19 07:26 mybackend-nsm_11/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  494M Dec 19 04:06 mybackend-nsm_12/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  109M Dec 19 05:53 mybackend-nsm_12/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs   36K Dec 19 04:06 mybackend-nsm_12/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  430M Dec 19 07:31 mybackend-nsm_13/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 07:31 mybackend-nsm_13/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  274M Dec 19 03:03 mybackend-nsm_14/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs     0 Dec 19 06:55 mybackend-nsm_14/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 03:03 mybackend-nsm_14/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  376M Dec 18 04:09 mybackend-nsm_15/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  8.0K Dec 19 09:10 mybackend-nsm_17/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  324M Dec 19 07:41 mybackend-nsm_18/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  640K Dec 19 07:41 mybackend-nsm_18/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs     0 Dec 19 05:53 mybackend-nsm_19/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs   28K Dec 19 06:22 mybackend-nsm_1/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs   24K Dec 19 06:50 mybackend-nsm_20/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  132M Dec 19 06:55 mybackend-nsm_21/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs     0 Dec 19 05:13 mybackend-nsm_21/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  314M Dec 19 05:31 mybackend-nsm_22/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs     0 Dec 19 08:16 mybackend-nsm_22/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 05:31 mybackend-nsm_22/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  450M Dec 19 06:42 mybackend-nsm_23/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs 1012K Dec 19 06:42 mybackend-nsm_23/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs     0 Dec 17 12:15 mybackend-nsm_24/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  471M Dec 17 12:14 mybackend-nsm_25/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  182M Dec 19 07:21 mybackend-nsm_25/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  500K Dec 17 12:14 mybackend-nsm_25/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  264M Dec 19 04:08 mybackend-nsm_26/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs   16K Dec 19 07:44 mybackend-nsm_26/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 04:08 mybackend-nsm_26/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  410M Dec 19 06:29 mybackend-nsm_27/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs     0 Dec 19 07:55 mybackend-nsm_27/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs   28K Dec 19 06:29 mybackend-nsm_27/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  314M Dec 19 07:21 mybackend-nsm_28/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  380M Dec 19 05:23 mybackend-nsm_29/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs   28K Dec 19 08:16 mybackend-nsm_29/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  380K Dec 19 05:23 mybackend-nsm_29/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  517M Dec 19 09:22 mybackend-nsm_2/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  772K Dec 19 09:22 mybackend-nsm_2/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  336M Dec 19 07:14 mybackend-nsm_30/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  668K Dec 19 07:14 mybackend-nsm_30/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs   14M Dec 19 07:32 mybackend-nsm_31/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs     0 Dec 19 06:30 mybackend-nsm_32/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  361M Dec 17 14:13 mybackend-nsm_4/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs   27M Dec 19 07:32 mybackend-nsm_4/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  292K Dec 17 14:13 mybackend-nsm_4/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  291M Dec 19 06:22 mybackend-nsm_5/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs     0 Dec 19 07:44 mybackend-nsm_5/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs   32K Dec 19 06:22 mybackend-nsm_5/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs   84M Dec 19 08:06 mybackend-nsm_7/tlogs/head.db.clone.tmp
-rw-r--r-- 1 ovs ovs  381M Dec 19 07:55 mybackend-nsm_8/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs   20K Dec 19 07:55 mybackend-nsm_8/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  267M Dec 19 05:23 mybackend-nsm_9/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 05:23 mybackend-nsm_9/tlogs/head.db.clone.wal
-rw-r--r-- 1 ovs ovs  223M Dec 19 07:53 voldrv/tlogs/head.db.clone
-rw-r--r-- 1 ovs ovs  4.0K Dec 19 07:53 voldrv/tlogs/head.db.clone.wal
```

**Comment by simonOpenV** [comment]
Fixed in #420 
Packaging info: openvstorage-health-check_3.6.5-dev.1516366275.09bee99-1_amd64.deb
--

**Comment by JeffreyDevloo** [comment]
Part of the https://github.com/openvstorage/openvstorage-health-check/releases/tag/3.6.5 release
